src/gui/editors/AutomationEditor.cpp

```diff
@@ -1627,11 +1627,11 @@ void AutomationEditor::wheelEvent(QWheelEvent * we )
 		int x = m_zoomingXModel.value();
 		if( we->delta() > 0 )
 		{
-			x--;
+			x++;
 		}
 		if( we->delta() < 0 )
```

Comment:
```c++
else if( we->delta() < 0 )
```

Will prevent one useless test when `x > 0`. I know it's not directly related to this PR, but spotted this so...